### ISSUE TITLE
Unable to start when placed in zeus (Both SFM+ & HeliSim)

### ISSUE THREAD

**Issue by xSlykiller** [AH-64D-Apache-Official-Project/AH-64D#639]
**Describe the bug**

If another player spawns the HeliSim version of the apache for me, when I enter, I cannot start the aircraft. The battery nor APU will start unless I place the apache down in zeus myself. 

**To Reproduce**

1. Have fren place "AH-64D Apache Longbow (HeliSim)"
2. Get into Pilot seat
3. Get angry when battery no turn on. 

**Expected behavior**

It is my expectation, that upon entering the AH-64D Apache Longbow (HeliSim) that I should be able to start the battery and begin start up procedures. 

**Environments where the bug occurs**

* Multiplayer single-crew

https://github.com/user-attachments/assets/c70eb9da-d78c-41ab-a0ed-87ac9053a315

**Comment by xSlykiller** [comment]
Update: This issue is in both the SFM+ and HeliSim versions.

**Comment by Ollieollieolllie** [comment]
Was this a product of the Different Versions you were running on your server that you reported having issues with?
if you're able to retest, that would be greatly appreciated

**Comment by xSlykiller** [comment]
> Was this a product of the Different Versions you were running on your server that you reported having issues with? if you're able to retest, that would be greatly appreciated

No, after previously figuring out there was a conflict, we set up an entirely new mission in eden and a new commandline for our dedicated server to completely remove the current release version of the AH-64D project. We only had the current steam dev branch loaded on the server and made sure we offloaded the release version from the individual clients to prevent any future conflicts

**Comment by BradMick** [comment]
Tested in multiplayer today, placed 4x aircraft with zeus was able to start each aircraft without issue.
